ansible_collections/arista/avd/roles/eos_designs/templates/interface_descriptions/loopback_interfaces/overlay-loopback.j2

```diff
@@ -1 +1 @@
-EVPN_Overlay_Peering
```

Comment:
try to refactor to use the `switch.loopback_ipv4_description`

Reply:
Fully implemented and documented neutral variable "overlay_loopback_description".